tests/driver_pir/main.c

```diff
@@ -64,7 +65,7 @@ int main(void)
 {
     puts("PIR motion sensor test application\n");
     printf("Initializing PIR sensor at GPIO_%ld... ", (long)PIR_GPIO);
-    if (pir_init(&dev, PIR_GPIO) == 0) {
+    if (pir_init(&dev, &pir_param) == 0) {
```

Comment:
as stated above, use params from pir_params here (change to &pir_params[0])

Reply:
done